app/models.py

```diff
@@ -0,0 +1,8 @@
+import dataclasses

```

Comment:
It is easier to `from dataclasses import dataclass` and use `@dataclass`, isn't it?

Reply:
Agree